java/src/main/java/com/logicalclocks/hsfs/engine/FeatureGroupEngine.java

```diff
@@ -102,6 +96,11 @@ public void saveFeatureGroup(FeatureGroup featureGroup, Dataset<Row> dataset,
 
     // Update the original object - Hopsworks returns the incremented version
     featureGroup.setVersion(apiFG.getVersion());
+    featureGroup.setId(apiFG.getId());
+    featureGroup.setStatisticsEnabled(apiFG.getStatisticsEnabled());
+    featureGroup.setCorrelations(apiFG.getCorrelations());
+    featureGroup.setHistograms(apiFG.getHistograms());
+    featureGroup.setStatisticColumns(apiFG.getStatisticColumns());
```

Comment:
statistics columns and statistics enabled should be the same as the original object

Reply:
yes you are right, backend should return the same as was sent, we can save setting it again.